app/views/recipes/index.html.erb

```diff
@@ -21,6 +21,7 @@
     <% end %>
   </div>
   <div id="recipes" class="flex-column flex-wrap justify-center lg:flex-nowrap">
+    <% if @recipes %>
```

Comment:
This shouldn't be necessary if @recipes is an array. Is it nil if there are no recipes for the user?

Reply:
Yes, it is nil if there are no recipes. I added that so I could see the application. The code broke when I added the scope because I hadn't implemented the scope correctly exeg(scope.where(user=user). So I got cannot read each of nil error. I know if @recipes is not needed. It's temporary.